src/platforms/linux/daemon/wireguardutilslinux.cpp

```diff
@@ -562,6 +582,55 @@ bool WireguardUtilsLinux::rtmSendRule(int action, int flags, int addrfamily) {
   return true;
 }
 
+bool WireguardUtilsLinux::rtmSendExclude(int action, int flags,
+                                         const QHostAddress& address) {
+  constexpr size_t fib_max_size =
+      sizeof(struct fib_rule_hdr) + RTA_SPACE(sizeof(struct in6_addr));
+
+  char buf[NLMSG_SPACE(fib_max_size)];
```

Comment:
If you use an initializer, you can avoid the memsets later.  ie char buf[...]={0};